src/common/NoResults/styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { ReactComponent as noresults } from "../../images/noresults.svg";
+
+export const StyledNoResults = styled(noresults)`
+    width: 668px;
+    height: 508px;
+    align-self: center;
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px) {
+        width: 68vw;
+        height: 54vw;
```

Comment:
It would be nice if we also had media query for mobile

Reply:
I think it looks great just like it is tbh